test/mock/createInstantSearch.ts

```diff
@@ -59,16 +59,10 @@ export const createInstantSearch = (
     removeListener: jest.fn(),
     on: jest.fn(),
     once: jest.fn(),
-    off: jest.fn(),
```

Comment:
these functions were declared on the type, but not actually on the events we had